### PR TITLE
migration fixes and db.js with tests

### PR DESCRIPTION
**Pull request by ZoeIML**

### CODE REVIEW

**Comment by ZoeIML** [comment]
Migrations only ready for review.

File: knexfile.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+// Update with your config settings.
```
Comment (don-smith) [inline knexfile.js]: This file should be in the `server/db` folder. When you pull from `development` you'll get the one that's already there.

File: migrations/20180503104549_orders.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+exports.up = (knex, Promise) => {
```
Comment (don-smith) [inline migrations/20180503104549_orders.js]: move the `migrations` folder into the `server/db` folder

File: migrations/20180503104549_orders.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+exports.up = (knex, Promise) => {
+  return knex.schema.createTable('orders', (table) => {
+    table.increments('id').primary()
+    table.int('date').default(Date.now())
+    table.boolean('is_complete')
+    table.int('owner_id').references()
```
Comment (don-smith) [inline migrations/20180503104549_orders.js]: This should be `table.integer('owner_id').references('users.id')`

File: migrations/20180503104733_order_items.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+exports.up = (knex, Promise) => {
+  return knex.schema.createTable('order_items', (table) => {
+    table.increments('id').primary()
+    table.int('user_id').references()
+    table.int('order_id').references()
```
Comment (don-smith) [inline migrations/20180503104733_order_items.js]: These 2 calls to `references()` should include a string to the associated `'table.primary_key'`.

File: seeds/test-seed-orderitems.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
+exports.seed = (knex, Promise) => {
```
Comment (don-smith) [inline seeds/test-seed-orderitems.js]: Let's move the `seeds` folder into the `server/db` folder

File: server/server.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+const path = require('path')
+const express = require('express')
+
+const server = express()
+
+const homeRoutes = require('./routes/home')
+
+server.use(express.static(path.join(__dirname, '../public')))
+server.use('api/v1/home', homeRoutes)
```
Comment (don-smith) [inline server/server.js]: This route should be `'/api/v1/current-order'`

File: server/db/seeds/test-seed-orderitems.js [hunk 7]
```diff
@@ -0,0 +1,23 @@
+exports.seed = (knex, Promise) => {
```
Comment (don-smith) [inline server/db/seeds/test-seed-orderitems.js]: let's rename this file to `orderitems.js`

File: server/db/seeds/test-seed-orders.js [hunk 8]
```diff
@@ -0,0 +1,21 @@
+exports.seed = (knex, Promise) => {
```
Comment (don-smith) [inline server/db/seeds/test-seed-orders.js]: rename this file to `orders.js`

**Pull request merged**